lib/Analysis/Utility.cpp

```diff
@@ -798,7 +798,6 @@ bool matchMFMAAndDotOperandShuffleCase(RankedTensorType srcTy,
   return dotOperandLayout.getParent() == mfmaLayout &&
          dotOperandLayout.getOpIdx() == 0 && mfmaLayout.getIsTransposed() &&
          dotOperandLayout.getKWidth() == 8 &&
-         getContigPerThread(mfmaLayout)[1] == 4 &&
```

Comment:
@antiagainst was this line necessary? mfma layout used to always return 4 when transposed.

Reply:
Yeah it should be fine to drop this line.